app/models/setting/content.rb

```diff
@@ -67,6 +67,8 @@ def self.load_defaults
                  false, N_('Restrict Composite Content View promotion')),
         self.set('check_services_before_actions', N_("Whether or not to check the status of backend services such as pulp and candlepin prior to performing some actions."),
                  true, N_('Check services before actions')),
+        self.set('foreman_proxy_content_auto_sync', N_("Whether or not to auto sync the smart proxys after a content view promotion."),
+                 true, N_('Sync smart proxy after cv promotion')),
```

Comment:
nitpick: I'd use "Content View" instead of cv here.  Also capitalize smart proxy and content view